src/modules/Dropdown/Dropdown.js

```diff
@@ -1058,8 +1058,8 @@ export default class Dropdown extends Component {
     return (
       <select type='hidden' aria-hidden='true' name={name} value={value} multiple={multiple}>
         <option value='' />
-        {_.map(options, option => (
-          <option key={option.value} value={option.value}>{option.text}</option>
+        {_.map(options, (option, i) => (
+          <option key={option.key || option.value} value={option.value}>{option.text}</option>
```

Comment:
Yep.  In fact, this should use a factory which already requires users to use keys when passing objects or elements.  We use literal values for the key otherwise.

See `factories.js` where we should likely add an HTML factory for `option`:
https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/lib/factories.js#L125